definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField.json

```diff
@@ -3639,6 +3639,13 @@
     "UserRole": "citizen",
     "CRUD": "CRU"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "solApplicationFeeInPounds",
+    "UserRole": "caseworker-divorce-solicitor",
+    "CRUD": "R"
+  },
```

Comment:
Suggestion: to add this to the nonprod file so that merging your PR to master would be easy. Also would just caseworker-divorce-solicitor be sufficient or adding 'R' also to the other roles would be necessary.